atomium/templates/pager/pager.component.inc

```diff
@@ -36,14 +36,14 @@ function atomium_preprocess_pager(array &$variables, $hook) {
   $element = $variables['element'];
   $quantity = $variables['quantity'];
   $items = array();
-  global $pager_page_array, $pager_total;
 
+  $pager_total_elem = $GLOBALS['pager_total'][$element];
```

Comment:
I would not use a new var there, I would use `$GLOBALS` directly, no ?

Reply:
It is just because it is being called 6 times.

Reply:
Ok fine.

Reply:
Actually I would use the same code as in `7.x-3.x`. If we don't, components in `7.x-2.x` and `7.x-3.x` might be different, and that, it's better to avoid.
We should also fix the missing `(int)` cast in the pager.